ansible-deployment/switch/tasks/switches.j2

```diff
@@ -0,0 +1,35 @@
+[ml2_ansible]
+coordination_uri=redis://:{{ masterauth_value.stdout }}@{{ master_ip }}:6379/
+[ansible:switch1]
+ansible_connection=ansible.netcommon.network_cli
+ansible_network_os=dellemc.os9.os9
+ansible_host={{ ansible_switch_host }}
+ansible_user=admin
+ansible_ssh_pass={{ ansible_switch_ssh_password }}
+ansible_become_password={{ ansible_switch_ssh_password }}
+ansible_become=yes
+ansible_become_method=enable
+mac=4c:d9:8f:dc:e0:c9
+manage_vlans=True
+ansible_connection=local
+stp_edge=True
+[ansible:brocade-15-42]
+ansible_connection=ansible.netcommon.network_cli
+ansible_network_os=nos
+ansible_host=10.0.15.42
+ansible_user=admin
+ansible_ssh_pass={{ ansible_brocade_ssh_password }}
+ansible_become=no
+mac=50:eb:1a:61:5d:6a
+manage_vlans=True
+ansible_connection=local
+stp_edge=True
+[ansible:cisco-23-40]
+ansible_network_os=nxos
+ansible_host=10.2.23.40
+ansible_user=admin
+ansible_ssh_pass={{ ansible_switch_ssh_password }}
```

Comment:
Just want to double check that this switch has the same password as switch1?

Reply:
Good catch - it does not, so this ansible variable should probably be given a different name.

Reply:
any suggestions for a variable name? `ansible_switch_cisco_23_40_ssh_password` or maybe I could shorten it to `ansible_cisco_23_40_pass`?

Reply:
I'm fine with both names.

Reply:
I'd suggest `ansible_cisco_23_40_password` for consistency with our extremely rigorous naming conventions for the other two switches, but I don't think it's a huge deal as long as it's documented somewhere.